test/docs/dash-test-workflow-saithrif.md

```diff
@@ -6,3 +6,64 @@ This document describes the DASH test workflow with SAI thrift.
 
 ![dash-test-wflow-saithrift](../images/dash-test-wflow-saithrift.svg)
 
+The figure above captures some important concepts:
```

Comment:
```suggestion
The figure above captures some important concepts:
```
The previous figure highlights the following test work-flow important concepts:

Reply:
How about:
The previous figure highlights the following important test work-flow concepts:

Reply:
Go ahead and make the change.